src/components/dialog/dialog.js

```diff
@@ -142,8 +132,18 @@ const VUIDialog = ({
             DOM.page.appendChild(DOM.backdrop);
         }
 
-        //  Add class to make dialog visible
+        //  Grabs elements that are focusable inside this dialog instance.
```

Comment:
Definitely a nit but does get to my OCD 😄. Most of our inline comments don't have 2 spaces after the two forward slashes but this one does, and the two below, for some reason.